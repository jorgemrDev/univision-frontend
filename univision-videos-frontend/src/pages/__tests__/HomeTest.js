import React from "react";
import Home from "../home";
import renderer from "react-test-renderer";

test("Home Loading correctly", () => {
  const component = renderer.create(<Home></Home>);
  let tree = component.toJSON();
  expect(tree.children.length).toBe(3);
  expect(tree).toMatchSnapshot();
});
