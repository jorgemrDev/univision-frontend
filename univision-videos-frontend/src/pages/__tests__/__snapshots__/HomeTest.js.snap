// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Loading correctly 1`] = `
<div
  className="ant-row"
  style={Object {}}
>
  <div
    className="ant-col ant-col-24"
    style={
      Object {
        "marginTop": 25,
        "textAlign": "center",
      }
    }
  >
    <h1
      style={
        Object {
          "fontSize": 35,
          "fontWeight": "bold",
        }
      }
    >
      Uploaded Videos
    </h1>
  </div>
  <div
    className="ant-col ant-col-24"
    style={Object {}}
  >
    <div
      className="loading"
    >
      <div
        className="ant-spin ant-spin-lg ant-spin-spinning"
      >
        <span
          className="ant-spin-dot ant-spin-dot-spin"
        >
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
        </span>
      </div>
    </div>
  </div>
  <div
    className="ant-col ant-col-24"
    style={Object {}}
  >
    <footer
      className="ant-layout-footer footer"
    >
      <p>
        Jorge Moreno R
      </p>
    </footer>
  </div>
</div>
`;

exports[`Render Loading correctly 1`] = `
<div
  className="ant-row"
  style={Object {}}
>
  <div
    className="ant-col ant-col-24"
    style={
      Object {
        "marginTop": 25,
        "textAlign": "center",
      }
    }
  >
    <h1
      style={
        Object {
          "fontSize": 35,
          "fontWeight": "bold",
        }
      }
    >
      Uploaded Videos
    </h1>
  </div>
  <div
    className="ant-col ant-col-24"
    style={Object {}}
  >
    <div
      className="loading"
    >
      <div
        className="ant-spin ant-spin-lg ant-spin-spinning"
      >
        <span
          className="ant-spin-dot ant-spin-dot-spin"
        >
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
          <i
            className="ant-spin-dot-item"
          />
        </span>
      </div>
    </div>
  </div>
  <div
    className="ant-col ant-col-24"
    style={Object {}}
  >
    <footer
      className="ant-layout-footer footer"
    >
      <p>
        Jorge Moreno R
      </p>
    </footer>
  </div>
</div>
`;
